src/services/user.js

```diff
@@ -308,9 +308,10 @@ const getImage = async username => {
 
 /**
  * Reset the current user's from preferred image to user's default image
+ * @returns {Response} Determines if the image was reset successfully
  */
 const resetImage = () => {
-  http.post('/profiles/image/reset', '');
+  return http.post('/profiles/image/reset', '');
```

Comment:
I only added a return function so that I could check to see if the post was successful. I don't actually parse the response but check to see that a response was received.